JAX 0.10.0 Tests were failed because of environmental var called `ROCPROFILER_QUEUE_INTERPOSITION`

## Summary

0.10.0 tests were failed yesterday for the nightly: https://github.com/ROCm/rockrel/actions/runs/28563094276 after the changes from https://github.com/ROCm/TheRock/pull/6309
```
+ unset ROCM_ROOT
+ unset HIP_DEVICE_LIB_PATH
+ unset JAX_ROCM_PLUGIN_INTERNAL_BITCODE_PATH
+ unset JAX_ROCM_PLUGIN_INTERNAL_LLD_PATH
+ unset HIP_VISIBLE_DEVICES
+ unset ROCR_VISIBLE_DEVICES
+ python -c 'import jax; print(jax.local_devices())'
F0702 06:00:51.762637     726 environment.cpp:94] No boolean value provided for ROCPROFILER_QUEUE_INTERPOSITION
*** Check failure stack trace: ***
/__w/_temp/3a863c94-fd28-4f84-8915-0ecc73c7fb38.sh: line 10:   726 Aborted                 (core dumped) python -c "import jax; print(jax.local_devices())"
Error: Process completed with exit code 134.
```

--- a/.github/workflows/test_linux_jax_wheels_partial.yml
+++ b/.github/workflows/test_linux_jax_wheels_partial.yml
@@ -265,9 +265,9 @@ jobs:
           # JAX 0.10.2 only: ROCm/HIP runtime tuning to avoid a pytest
           # slowdown/hang; other releases keep the runtime defaults.
           # TODO:(magaonka-amd) remove these as soon as fixes from corresponding system teams lands.
-          ROCPROFILER_QUEUE_INTERPOSITION: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '0' || '' }}
-          DEBUG_HIP_DYNAMIC_QUEUES: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '0' || '' }}
-          HSA_NO_SCRATCH_RECLAIM: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '1' || '' }}
+          ROCPROFILER_QUEUE_INTERPOSITION: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '0' }}
+          DEBUG_HIP_DYNAMIC_QUEUES: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '0' }}
+          HSA_NO_SCRATCH_RECLAIM: ${{ inputs.jax_ref == 'rocm-jaxlib-v0.10.2' && '1' }}
 
           # Reduce compilation/autotune memory pressure.
           XLA_FLAGS: >-

